static/app/utils/theme/theme.tsx

```diff
@@ -1772,7 +1761,7 @@ const lightThemeDefinition = {
   type: 'light' as 'light' | 'dark',
   // @TODO: color theme contains some colors (like chart color palette, diff, tag and level)
   ...commonTheme,
-  fontSize,
```

Comment:
Note that the `fontSize` property of `commonTheme` was being overwritten here. That is no longer the case, but the overrides match the new values. No functional change here.